docs/UserGuide.md

```diff
@@ -375,28 +505,49 @@ Format: `create-task tk/TASKDESCRIPTION [REMARKS]`
 * Tasks may have no dates, a single date indicating a deadline, or two dates to define a start and end period.
 * The dates can be specified under `REMARKS` with the format `d/YYYY-MM-DD`
 * Tasks are unique in WedLinker, there would not be any duplicated tasks.
-* Tasks can be assigned to a contact using the [assign-task](#assigning-a-task-to-a-contact--assign-task-) command.
+* Tasks can be assigned to a contact using the [assign-task](#assigning-a-task-to-a-contact-assign-task-) command.
+
+<box type="warning" seamless>
 
-### Delete a Task : `delete-task`
+<details open>
+<summary><strong>Caution:</strong></summary>
+<br>
+
+- Tasks with single or multiple spaces in their names are considered distinct and treated as separate tasks. For example:
+    - <span class="preserve-whitespace">Send invitations vs. Send  invitations</span>
+</details>
+</box>
+
+##### Delete a Task : `delete-task`
 
 Deletes a `Task` from WedLinker
 
 Format: `delete-task INDEX`
 
-* Deletes the specific `Task` at the INDEX when in [list-tasks](#listing-all-tasks--list-tasks) view.
+* Deletes the specific `Task` at the INDEX when in [list-tasks](#listing-all-tasks-list-tasks) view.
 * The index **must be a positive integer** 1, 2, 3, …​.
 
-### Assigning a Task to a contact : `assign-task` 
+##### Assigning a Task to a contact : `assign-task` 
 
 Format: `assign-task PERSONINDEX TASKINDEX`
 
 * Assigns a `Task` to a contact.
-* The indexes correspond to the indexes when in the [list-tasks](#listing-all-tasks--list-tasks) view.
+* The indexes correspond to the indexes when in the [list-tasks](#listing-all-tasks-list-tasks) view.
 * The `PERSONINDEX` refers to the index of the person shown under the **People** column.
 * The `TASKINDEX` refers to the index of the task shown under the **Tasks** column
 * The indexes **must be positive integers** 1, 2, 3, …​.
 
-### Un-assigning a Task from a contact : `unassign-task` 
+<box type="warning" seamless>
+<details open>
+<summary><strong>Caution:</strong></summary>
+<br>
+
+- Trying to assign multiple tasks to a person in a singular command will cause only 1 of those tasks (at random) to be assigned. For example:
```

Comment:
Our assign-task command takes in multiple task inputs! It will assign all the later task indices. This is documented in our user message (not fully, need to fix this) just not in the UG. So, `atask 1 1 2 3` will assign all of those tasks (1 2 3) to be assigned to person 1 if they are a vendor.

![Screenshot 2024-11-11 at 12 52 17 PM](https://github.com/user-attachments/assets/7dc45c24-705c-40e4-b065-68d8b9e76d9f)
*Evidence of implementation and feature intent in description of command*

But also requires an update of the UI text for the command format, to put TASK_INDEX... with the ellipses - I am pretty sure that this is allowed under tweak UI text in FAQ (below) because the current text is incorrect in that it implies that only 1 task index can be specified.
 
<img width="808" alt="Screenshot 2024-11-11 at 12 54 15 PM" src="https://github.com/user-attachments/assets/370823c4-b2a8-4d20-92c0-514d58f07ab2">